app/scripts/models/auth_brokers/redirect.js

```diff
@@ -86,14 +87,17 @@ define([
       // event handlers before the flow completes.
       var self = this;
 
-      return p().delay(100).then(function () {
-        if (self.isOriginalTab() || self.canVerificationRedirect()) {
-          return self.finishOAuthSignUpFlow(account)
-            .then(function () {
-              return new HaltBehavior();
-            });
-        }
-      });
+      return proto.afterCompleteSignUp.call(self, account)
```

Comment:
I noticed several brokers do not call the prototype function to get the default behavior. This should be fixed in a follow up issue.